ui/app/components/policy-form.hbs

```diff
@@ -70,36 +82,6 @@
         <span class="is-size-9 has-text-grey has-bottom-margin-l">
           You can use Alt+Tab (Option+Tab on MacOS) in the code editor to skip to the next field.
         </span>
-        {{! Only renders button (and modal) if not already in the "create policy" modal }}
```

Comment:
These links were incorrect - this copy and correct links already render below the example policy in the modal 
<img width="933" alt="Screenshot 2023-09-27 at 5 39 23 PM" src="https://github.com/hashicorp/vault/assets/68122737/66f613e2-b5d1-4c46-9a1e-13d14ee401b2">